java/vector/src/test/java/org/apache/arrow/vector/types/pojo/TestExtensionType.java

```diff
@@ -17,6 +17,9 @@
 
 package org.apache.arrow.vector.types.pojo;
 
+import static junit.framework.TestCase.assertTrue;
```

Comment:
nit: This is a JUnit 3 method - while it seems to be used elsewhere, it would be preferable to use `org.junit.Assert.assertTrue` instead.

Reply:
thanks a lot, fixed :)